package.json

```
@@ -28,7 +28,11 @@
     "clean": "rimraf dist",
     "prepublishOnly": "run-s clean build"
   },
+  "peerDependencies": {
+    "@shopify/hydrogen": "^1.0.0"
```

any reason the peer dependency isn't `>= 1.0.0`?

```suggestion
    "@shopify/hydrogen": ">= 1.0.0"
```